lib/components/app/app-menu.js

```diff
@@ -105,4 +231,33 @@ const mapDispatchToProps = {
   toggleMailables: callTakerActions.toggleMailables
 }
 
-export default withRouter(connect(mapStateToProps, mapDispatchToProps)(AppMenu))
+export default withRouter(
+  connect(
+    mapStateToProps,
+    mapDispatchToProps
+  )(injectIntl(AppMenu))
+)
+
+/**
+ * Renders a label and icon either from url or font awesome type
+ */
+const IconAndLabel = injectIntl(({ iconType, iconUrl, intl, label }) => {
```

Comment:
Last minute catch: in a functional component, use the `const intl = useIntl()` hook in the method body instead.

Reply:
Good call! They really did think of everything

Updated in cf487ffe27cc35053cdbd417b96dbaa45691faad